src/controller_examples/zookeeper_controller/common/mod.rs

```diff
@@ -8,31 +8,60 @@ verus! {
 #[is_variant]
 pub enum ZookeeperReconcileStep {
     Init,
-    AfterGetHeadlessService,
-    AfterCreateHeadlessService,
-    AfterUpdateHeadlessService,
-    AfterGetClientService,
-    AfterCreateClientService,
-    AfterUpdateClientService,
-    AfterGetAdminServerService,
-    AfterCreateAdminServerService,
-    AfterUpdateAdminServerService,
-    AfterGetConfigMap,
-    AfterCreateConfigMap,
-    AfterUpdateConfigMap,
+    AfterKRequestStep(ActionKind, SubResource),
+    // AfterGetHeadlessService,
```

Comment:
I guess we can remove these commented code now?

Reply:
Updated